pulseapi/entries/models.py

```diff
@@ -23,17 +23,17 @@ class Entry(models.Model):
     A pulse entry
     """
     title = models.CharField(max_length=140)
-    description = models.CharField(max_length=300)
-    content_url = models.URLField()
-    thumbnail_url = models.URLField()
+    description = models.CharField(max_length=300, blank=True)
+    content_url = models.URLField(blank=True)
```

Comment:
Are we sure about this one? Are people supposed to be able to post entries without a URL to point to the thing the entry was supposed to be for?